packages/plugins/scripts-runner/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "embark-scripts-runner",
```

Comment:
Very minor comment, but I think `script-runner` sounds better. What does the English master, @michaelsbradleyjr , think?

Reply:
I think it could go either way. Since it may often be running more than one script, I can see a good case for it being `scripts-runner`.